ampl-converter/src/main/java/com/powsybl/ampl/converter/AmplNetworkWriter.java

```diff
@@ -1339,12 +1448,23 @@ private void writeShunts(AmplExportContext context) throws IOException {
                 int vlNum = mapper.getInt(AmplSubset.VOLTAGE_LEVEL, t.getVoltageLevel().getId());
                 double vb = t.getVoltageLevel().getNominalV();
                 double zb = vb * vb / AmplConstants.SB;
-                double b1 = 0;
-                double b2 = sc.getModel(ShuntCompensatorLinearModel.class).getBPerSection() * sc.getMaximumSectionCount() * zb;
+                double b1;
+                double b2;
+                int points;
+                int sectionCount = 1;
```

Comment:
I would move the initialization inside the if block.

Reply:
Ok!